### DISCUSSION TITLE
Using entire x-axis without buffer at both end

### DISCUSSION THREAD

**Discussion by Ho-ChunHuang-NOAA** [dtcenter/METplus#1545] [Plot Generation] [answered]
Case 20220401_131024

There is a whiteout area on both end of Axis.  How can I  make the plotting area use the entire x-axis?

**Comment by Ho-ChunHuang-NOAA** [comment]
Correction, on both end of x-axis

**Reply by TatianaBurek** [reply]
Currently it is not possible to accomplish using Python. 
I created a GitHub issue to add this feature to METviewer/METplotpy.
https://github.com/dtcenter/METplotpy/issues/217
The fix will be available in the next release.

**Comment by Ho-ChunHuang-NOAA** [comment, marked as answer]
Hi, Tatiana


Thanks, I will test whether I can get the plot with
<execution_type>Rscript</execution_type>

Before it is available in python

Ho-Chun Huang, Ph.D.

IMSG at NOAA/NWS/NCEP/EMC

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958

**Reply by TatianaBurek** [reply]
@Ho-ChunHuang-NOAA Were you able to get a plot with 
`<execution_type>Rscript</execution_type>`
Let me know if you need more assistance

**Comment by Ho-ChunHuang-NOAA** [comment]
Please close the ticket.  I will stick with the Python script and live with
the buffer zone in the time series plot.

Ho-Chun Huang, Ph.D.

IMSG at NOAA/NWS/NCEP/EMC

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958
